x.py

```diff
@@ -131,6 +131,36 @@ def cppcheck(args):
 
     run([command, *options, *sources])
 
+def package_source(args):
+    version = args.release_version.strip()
+    if SEMVER_REGEX.match(version) is None:
+        raise RuntimeError(f"Kvrocks version should follow semver spec, got: {version}")
+    
+    # 0. Write input version to VERSION file
+    with open('VERSION', 'w+') as f:
+        f.write(version)
```

Comment:
I think we can just discard this file and pass the version to cmake directly, how do you think about this?